crates/core/thin_edge_json/src/health.rs

```diff
@@ -0,0 +1,27 @@
+use std::process;
+
+use mqtt_channel::{Message, PubChannel, Topic};
+use serde_json::json;
+use time::OffsetDateTime;
+
+pub fn health_check_topics(daemon_name: &str) -> Vec<String> {
```

Comment:
I would return a  `TopicFilter` here  adding a `.try_into().expect("health check topics must be valid")`.

Reply:
yes, make sense. Changed it accordingly.